demos/bundlers/build.mjs

```diff
@@ -0,0 +1,139 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import url from 'node:url';
+
+import { rollup } from 'rollup';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import webpack from 'webpack';
+
+const dirname = path.dirname(url.fileURLToPath(import.meta.url));
+const tmpDir = path.join(dirname, 'tmp');
+
+const inputs = [
+  `${dirname}/test/import-default.js`,
+  `${dirname}/test/import-named.js`,
+  `${dirname}/test/import-indirect.js`,
+  `${dirname}/test/require-default.cjs`,
+  `${dirname}/test/require-indirect.cjs`
+];
+
+const htmlTemplate = `<!DOCTYPE html>
+<html>
+<head>
+<title>{{title}}</title>
+<link rel="stylesheet" href="../node_modules/qunit/qunit/qunit.css">
+{{scriptTag}}
+</head>
+<body>
+<div id="qunit"></div>
+</body>
+</html>
+`;
+
+// Rollup configuration
+const rollupOutputs = [
+  {
+    dir: tmpDir,
+    entryFileNames: '[name].[format].js',
+    format: 'es'
+  },
+  {
+    dir: tmpDir,
+    entryFileNames: '[name].[format].js',
+    format: 'cjs'
+  },
+  {
+    dir: tmpDir,
+    entryFileNames: '[name].[format].js',
+    format: 'iife'
+  },
+  {
+    dir: tmpDir,
+    entryFileNames: '[name].[format].js',
+    format: 'umd',
+    name: 'UNUSED'
```

Comment:
What is this about?

Reply:
When reading a file as ESM, Rollup differentiates between a file that has no exports (i.e. app code), and a file that has exports. When reading a file as CJS, Rollup doesn't understand this, and so it insists on exporting. When using `umd`, one of the branches assigns a global to the overall `exports` which requires a name. Without the build will fail.

The exports is an empty object, but Rollup wants to put it somewhere, so gave it the name `window.` `UNUSED`.

Reply:
A comment in code could be useful here.

Reply:
Ack. Done in https://github.com/qunitjs/qunit/pull/1791.